src/commands/device/index.ts

```diff
@@ -77,58 +77,67 @@ export default class DeviceCmd extends Command {
 
 		const balena = getBalenaSdk();
 
+		const jsonExpand = {
+			$expand: {
+				device_tag: {
+					$select: ['tag_key', 'value'],
+				},
+				...expandForAppName.$expand,
+			},
+		} as const;
+
+		const jsonComputedFieldsSelect = {
+			$select: [
+				'overall_status',
+				'overall_progress',
+				'should_be_running__release',
+			],
+		} as const;
+
+		const tableParams = {
+			$select: [
+				'device_name',
+				'id',
+				'overall_status',
+				'is_online',
+				'ip_address',
+				'mac_address',
+				'last_connectivity_event',
+				'uuid',
+				'supervisor_version',
+				'is_web_accessible',
+				'note',
+				'os_version',
+				'memory_usage',
+				'memory_total',
+				'public_address',
+				'storage_block_device',
+				'storage_usage',
+				'storage_total',
+				'cpu_usage',
+				'cpu_temp',
+				'cpu_id',
+				'is_undervolted',
+			],
+			...expandForAppName,
+		} as const;
+
 		let device: ExtendedDevice;
 		if (options.json) {
 			const [deviceBase, deviceComputed] = await Promise.all([
-				balena.models.device.get(params.uuid, {
-					$expand: {
-						device_tag: {
-							$select: ['tag_key', 'value'],
-						},
-						...expandForAppName.$expand,
-					},
-				}),
-				balena.models.device.get(params.uuid, {
-					$select: [
-						'overall_status',
-						'overall_progress',
-						'should_be_running__release',
-					],
-				}),
+				balena.models.device.get(params.uuid, jsonExpand),
+				balena.models.device.get(params.uuid, jsonComputedFieldsSelect),
```

Comment:
Was there a reason to not keep the param objects inline like they were before?
It feels a bit harder to follow now that we have to scroll up to check what's fetched.

Reply:
I was mostly following the (new) pattern, but agree here and reverted